### DISCUSSION TITLE
Just a question (not important i guess lol)

### DISCUSSION THREAD

**Discussion by imnotcreatv** [maximmax42/Discord-CustomRP#172] [Q&A]
Is it possible that you can choose time zones? Because it can be confusing if you have to enter a time in the future for CRP to show "xx:xx elapsed" 

Sorry if its not really understandable haha Im not exactly the best in english

Anyways thanks in advance for answering (/helping)
~Fynn

**Comment by maximmax42** [comment]
To show how much "elapsed" you need to input time in the past, not future. If you set time in the future it shows how much is remaining. As for timezone, I believe it should set the time based on your own timezone.

**Reply by imnotcreatv** [reply]
Yeah no that exactly is the problem, (idk if its just germany but) its on summertime and not wintertime so I have to enter everything an hour later haha

But thanks for helping anyways :)
~Fynn

**Reply by maximmax42** [reply]
Weird. It should pull your actual time.
